src/js/services/wallet-history.service.js

```diff
@@ -22,6 +22,9 @@
       var service = {
         getCachedTxHistory: getCachedTxHistory,
         updateLocalTxHistoryByPage: updateLocalTxHistoryByPage,
+        processNewTxs: processNewTxs,
```

Comment:
Is this just a quick fix that is supposed to be changed later? Seems to me that the proper way to do this is to have an addTxToCache() method rather than exposing 3 methods. This would also reduce duplication in the controllers.